pkg/apb/types.go

```diff
@@ -62,7 +62,7 @@ type Plan struct {
 type Spec struct {
 	ID          string                 `json:"id"`
 	FQName      string                 `json:"name" yaml:"name"`
-	Image       string                 `json:"image"`
+	Image       string                 `json:"image,omitempty"`
```

Comment:
I don't understand the need for this change. I don't think it will hurt but don't know why we need it. 

I could see you doing `yaml:"-"` so that we never take anything for the APB spec.

Reply:
you are correct. I don't believe I fully understand yet what the rules and side-effects are for struct tags. I couldn't find anything that detailed `yaml:"-"` or the json/yaml struct tags in general, if there is anything you could point me to that would be helpful.